CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## Unreleased
 ### Changed
 - Switched from Glide to Go Modules.
+- Basic linting setup via a Makefile.
```

Comment:
This doesn't need a changelog entry because it doesn't affect users in any way.

Reply:
I think I've seen both approaches, but agreed, makes sense.